src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
```

Comment:
Consider using separate variables for the `color` and `side` before passing them to the constructor for better code readability.